includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -299,14 +288,11 @@ protected static function get_theme_features_config( $theme_slug ) {
 						'//style[ @id = "twentyeleven-header-css" ]',
 						'//link[ @id = "dark-css" ]',
 					],
-					'enable_determine_hero_images_transformer' => [],
 				];
 
 			// Twenty Ten.
 			case 'twentyten':
-				return [
-					'enable_determine_hero_images_transformer' => [],
-				];
+				return [];
```

Comment:
I suppose the `twentyten` case can be removed then?

Reply:
I tried removing but then I got warnings:

```
[15-Apr-2021 03:16:16 UTC] PHP Warning:  array_merge(): Expected parameter 2 to be an array, null given in /app/public/content/plugins/amp/includes/sanitizers/class-amp-core-theme-sanitizer.php on line 495
[15-Apr-2021 03:16:16 UTC] PHP Warning:  Invalid argument supplied for foreach() in /app/public/content/plugins/amp/includes/sanitizers/class-amp-core-theme-sanitizer.php on line 499
```

So it's currently needed, due to how that array is used in the `get_theme_features` method.